packages/delete/src/components/InfoPart/InfoPart.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import {Typography, Link} from 'tocco-ui'
+import PropTypes from 'prop-types'
+
+import LinkPopOver from './LinkPopOver'
+import {relatedPropTypes} from '../../utils/deleteRequestParser'
+
+const InfoPart = ({entityName, entityModel, primaryPks, relatedEntities, maxCountLink}) => {
+  const primaryTql = 'IN(pk,' + primaryPks.join(',') + ')'
+  return <Typography.Span>
+    <Typography.B>
+      {entityName} (<Link href={`/e/${entityModel}/list?tql=${primaryTql}`} rel="noopener noreferrer" target="_blank">
+        {primaryPks.length}
+      </Link>)
+    </Typography.B>
+    {
+      Object.keys(relatedEntities).length > 0
+      && <>
+        <span> / </span>
+        {Object.keys(relatedEntities).map(entityModel => {
+          const relatedEntity = relatedEntities[entityModel]
+          const tql = 'IN(pk,' + relatedEntity.pks.slice(0, maxCountLink).join(',') + ')'
+          const linkText = [...relatedEntity.pks, ...relatedEntity.pksOtherBu].length
+
+          const Count = relatedEntity.pks.length > 0
+            ? <Link href={`/e/${entityModel}/list?tql=${tql}`} rel="noopener noreferrer" target="_blank">
+              {linkText}
+            </Link>
+            : <Typography.Span>{linkText}</Typography.Span>
+
+          const Content = <LinkPopOver relatedEntity={relatedEntity} maxCountLink={maxCountLink}>{Count}</LinkPopOver>
+
+          return <React.Fragment
+            key={'entity-info-' + entityModel}>
+            {relatedEntity.entityName} ({Content})
+          </React.Fragment>
+        }).reduce((prev, curr) => [prev, ', ', curr])}
+      </>
+    }
+
+  </Typography.Span>
+}
+
+InfoPart.propTypes = {
+  entityName: PropTypes.string,
+  entityModel: PropTypes.string,
```

Comment:
From the usage I assumed entityName was the localized name of the entityModel (It's displayed in the bold tag on line 12)

Reply:
yes i know. it's a bit of a mess
I think from now on forward we should use
entityName / name for technical name e.g. User
entityLabel / label for the localized name e.g. Person
entityModel / model should be reserved for the model object (that contains the name and so on)
And entityKey / key for primary keys and avoid "id" or "pk" as substitute

entityId ist an object containing entityName and entityKey

what do you think? I could write it down in read the docs. but i think this will trigger some major changes. e.g. the selection right now looks like this:

```
{
  type: 'ID',
  ids: [....]
}

```
but should be renamed to 

```
{
 type: 'KEY',
 keys: []
}
```